skate-plugin/src/main/kotlin/com/slack/sgp/intellij/RepoAuthHotfix.kt

```diff
@@ -0,0 +1,28 @@
+/*
```

Comment:
This changes needs to be made in artifactory-authenticator

Reply:
Okay: 3f2536b386ee924943dc38193c756320d4f9981f